.env

```diff
@@ -56,4 +56,4 @@ SENTRY_DSN=
 IPC_AUTHKEY=ipc
 IPC_HOST=0.0.0.0
 IPC_PORT=6650
-WORKER_COUNT=8
+WORKER_COUNT=8
```

Comment:
Can you revert to the original version for this file too?

Reply:
Yep !